modules/javafx.graphics/src/main/java/com/sun/javafx/tk/PlatformImage.java

```diff
@@ -97,4 +100,6 @@ public void setPixels(int x, int y, int w, int h,
 
     public void setPixels(int dstx, int dsty, int w, int h,
                           PixelReader reader, int srcx, int srcy);
+
+    public void bufferDirty(Rectangle rect);
```

Comment:
Is there a policy about adding methods to interfaces? 1. add a default method or 2. add implementations to all existing implementations of the interface? 
In this case, we know all implementations, so I'm ok with no default method.

Reply:
Yes, since this is an internal interface (not in a public package) there is no problem adding it.